pkg/server/routes.go

```diff
@@ -10,11 +10,19 @@ import (
 
 func (s *server) routes(
 	authorizationMiddleware func(handler http.Handler) http.Handler,
-	corsMiddleware func(handler http.Handler) http.Handler) {
+	corsMiddleware func(handler http.Handler) http.Handler,
```

Comment:
Depending on how many middlewares we have over time, we may want to pass these differently. Something to think about.